### PR TITLE
Update Typescript support for common msgs 

### PR DESCRIPTION
**Pull request by Chrismacolor**
## Motivation
In response #2

## Description
- [x] Remove old message interfaces
- [x] Add ts-gen config file and correct package.json, add build scripts
- [x] Remove namepace prefixes on generator config
- [x] Add clear documentation for anyone to run generator and publish package
- [x] Add vision and geometry msgs and generate new messages

## How Was This Tested
Local development, Ubuntu 22.04.3 LTS

### CODE REVIEW

File: publishing.md
Comment (Chrismacolor) [inline publishing.md:1]: @entire Instructions for using and updating package

File: ros-ts-generator-config.json
Comment (Chrismacolor) [inline ros-ts-generator-config.json:1]: @entire This is where some of the manual work comes in, when adding new msg newspaces

File: publishing.md
Comment (entire) [reply publishing.md:1]: rad

File: ros-ts-generator-config.json
Comment (entire) [reply ros-ts-generator-config.json:1]: nice

**Review by entire — APPROVED** [review]
looks great lfg

**Comment by entire** [comment]
lets merge it and publish

**Pull request merged**